tests/functional/pod_and_daemons/test_standalone_noobaa_pod_node_affinity.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.ocs.node import (
+    apply_node_affinity_for_noobaa_pod,
+)
+from ocs_ci.ocs.resources.pod import (
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.framework.pytest_customization.marks import brown_squad
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+class TestNoobaaPodNodeAffinity:
+    @pytest.fixture(scope="session", autouse=True)
+    def teardown(self, request):
+        def finalizer():
```

Comment:
add an empty line above this and add doc strings

Reply:
Added doc string